cts/valgrind-pcmk.suppressions

```diff
@@ -262,7 +294,30 @@
 }
 
 {
-   glib - this is not funny anymore
+   glib - calloc/g_malloc 3
+   Memcheck:Leak
+   fun:calloc
+   fun:g_malloc0
+   obj:*/libgobject-*
+   obj:*/libgobject-*
+   fun:call_init.part.0
+   fun:_dl_init
+}
+
+{
+   dear glib - calloc/gmalloc 4
```

Comment:
forgot a "dear" :)

Reply:
Whoops.  Fixed.

Also, suppression files support '...' so I can actually make this file a lot smaller (and probably also suppress more crud at the same time).

Reply:
yeah the only concern is making sure the ... doesn't catch any pacemaker code

Reply:
Updated.  It may be easier to review the suppressions file as a whole instead of looking at the diff.